src/api/student.js

```diff
@@ -15,4 +16,26 @@ app.get('/', (req, res) => {
     res.successJson(student);
 });
 
+app.post('/', (req, res) => {
+    let workbook = xlsx.read(req.body.data, {type: 'binary'});
+    let studentJSON = xlsx.utils.sheet_to_json(workbook.Sheets[workbook.SheetNames[0]], {range:1});
+    for (let key in studentJSON) {
+        if (studentJSON.hasOwnProperty(key)) {
+            let s = studentJSON[key];
+            let student = {
+                id: s['Student ID'],
+                name: s['Student'],
+                status: s['Status'],
+                email: s['E-mail'],
+                major: s['Major']
+            };
+            if (StudentStore.isCurrentStudent(student) && StudentStore.isNewStudent(student)) {
+                addAction('NEW_STUDENT', student);
```

Comment:
They said that it is possible to have multiple excel documents instead of just one massive one containing all students. Therefore the doc that we get is not a master list where we can delete students who are not on it. 
In regard to "we do not want graduates", the store now has a `isCurrentStudent` which will be able to tell if the student is currently attending or not. That can be used else where to make sure the student is currently attending.